admin/data/parcelpro-shipping-settings-fields.php

```diff
@@ -125,6 +125,34 @@
         'description' => 'Customer note (billing address) invullen bij het veld "inhoud" in het verzendsysteen.',
         'desc_tip'    => true
     ),
+    'parcelpro_forecast_settings_postnl_show_delivery_date' => array(
+        'type'     => 'checkbox',
+        'title'    => 'Verwachte levertijd inschakelen (PostNl)',
+        'label'    => 'Verwachte levertijd inschakelen',
```

Comment:
Hieruit is het niet per sé duidelijk wat deze instelling doet: namelijk de verwachte levertijd weergeven _in de checkout_. Mogelijk is het handig om de `title` te veranderen naar: "Verwachte PostNL levertijd weergeven in checkout". De `label` zou ik korter laten, en hoeft iig niet de hele titel te dupliceren. Ik denk dat "Inschakelen" daarvoor genoeg is.